README.md

```diff
@@ -1,51 +1,56 @@
 listed in an aggregator.
 
+## Installation
 
-## Requirements
+```
+pip install gtfs-aggregator-checker
+```
 
-* `.env` - Acquire an [api key from transitland][1] and save it to a `.env` file
-  like `TRANSITLAND_API_KEY=SECRET`. Alternatively you can prefix commands with
-  the api key like `TRANSITLAND_API_KEY=SECRET python feed_checker.py [...]`.
+## Configure
 
-* `agencies.yml` - This file can have any structure as the feed checker just
-  looks for any urls (strings starting with `'http://'`), but the intended usage
-  is a [Cal-ITP agencies.yml file][2]. (to run the program without an
-  `agencies.yml` file, see the "Options" section below)
+The following env variables can be set in a `.env` file, set to the environment,
+or inline like `TRANSITLAND_API_KEY=SECRET python -m gtfs_aggregator_checker`.
 
-## Getting Started
+* `TRANSITLAND_API_KEY` An [api key from transitland][1].
 
-To install requirments and check urls run the following. The first time you run
-this it will take a while since the cache is empty.
+* `GTFS_CACHE_DIR` Folder to save cached files to. Defaults to
+`~/.cache/gtfs-aggregator-checker`
 
-``` bash
-pip install -r requirements.txt
-python feed_checker.py
-```
+## Getting Started
 
-The final line of stdout will tell how many urls were in `agencies.yml` and how
-many of those were matched in a feed. Above that it will list the domains for
-each url (in alphabetical order) as well group paths based on if the path was
-matched (in both `agencies.yml` and aggregator), missing (in `agencies.yml` but
-not aggregator) or unused (in aggregator but not in `agencies.yml`). An ideal
-outcome would mean the missing column is empty for all domains.
+## CLI Usage
 
+`python -m gtfs_aggregator_checker [YAML_FILE] [OPTIONS]`
 
-## CLI Usage
+`python -m gtfs_aggregator_checker` or `python -m gtfs_aggregator_checker
+/path/to/yml` will search a [Cal-ITP agencies.yml file][2] for any urls and see
+if they are present in any of the feed aggregators. Alternatively you can use a
+`--csv-file` or `--url` instead of an `agencies.yml` file.
 
-`python feed_checker.py` or `python feed_checker.py /path/to/yml` will search a
-[Cal-ITP agencies.yml file][2] for any urls and see if they are present in any
-of the feed aggregators.
+The final line of stdout will tell how many urls were in `agencies.yml` and how
+many of those were matched in a feed.
 
 ### Options
-* `python feed_checker.py --help` print the help
-* `--csv-file agencies.csv` load a csv instead of a Cal-ITP agencies yaml file (one url per line)
-* `--url http://example.com` Check a single url instead of a Cal-ITP agencies yaml file
+* `python -m gtfs_aggregator_checker --help` print the help
+* `--csv-file agencies.csv` load a csv instead of a Cal-ITP agencies yaml file
+  (one url per line)
+* `--url http://example.com` Check a single url instead of a Cal-ITP agencies
+  yaml file
 * `--verbose` Print a table of all results (organized by domain)
```

Comment:
This is no longer what the `--verbose` argument does. Can this documentation be updated?